src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -600,9 +561,12 @@ ucs_status_t uct_dc_mlx5_ep_get_bcopy(uct_ep_h tl_ep,
     UCT_DC_MLX5_IFACE_TXQP_GET(iface, ep, txqp, txwq);
     uct_rc_mlx5_ep_fence_get(&iface->super, txwq, &rkey, &fm_ce_se);
 
-    uct_dc_mlx5_iface_bcopy_post(iface, ep, MLX5_OPCODE_RDMA_READ, length,
-                                 remote_addr, rkey, desc, fm_ce_se, 0,
-                                 desc + 1, NULL);
+    uct_mlx5_txqp_bcopy_post(&iface->super, UCT_IB_QPT_DCI, txqp, txwq,
```

Comment:
i think we can still keep uct_dc_mlx5_iface_bcopy_post, but just as wrapper over uct_mlx5_txqp_bcopy_post